services/auth_service.py

```
@@ -16,29 +18,47 @@ def verify_password(plain_password : str, hashed_password :str):
-    return user, *create_access_token(user["user_id"])
+            user=UserInternal(
+                user_id=user_id,
+                email=data.email,
+                nickname=data.nickname,
+                password=hashed_password,
+                profile_image_url=data.profile_image_url,
+                created_at=user_creat_at
+            )
+        con.commit()
+        token, expires = create_access_token(user.user_id)
+        return user, token, expires
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)